packages/story-editor/src/components/library/panes/media/local/mediaEditDialog.js

```diff
@@ -141,6 +150,7 @@ function MediaEditDialog({ resource, onClose }) {
       secondaryText={__('Cancel', 'web-stories')}
       onPrimary={updateMediaItem}
       primaryText={__('Save', 'web-stories')}
+      dialogBoxStyleOverride={altText.length >= 21 ? styledDialogBox : null}
```

Comment:
So, I'm looking at the screenshot of the reported bug and what we've got here now and I think we've lost the display version of the media title. 

| Reported bug | Main now | 
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/10720454/144940477-3a48d606-2ca1-4647-a30c-f63cda4e295c.png) | <img width="573" alt="Screen Shot 2021-12-06 at 4 42 33 PM" src="https://user-images.githubusercontent.com/10720454/144940498-19cc8ef7-9f24-4d6a-bc93-c5d1b4dd6a17.png"> |

With that in mind, I'm curious what you think about just updating the input here to have an ellipsis for text-overflow and leaving the width as is. The title isn't getting displayed as display text anymore so there's nothing there to add text wrap to.

Reply:
Yeah, I am for it.

Reply:
Though. The title was removed so now only the input is what is being cut off. You can see the whole name of the image, you just have to scroll. If we throw in some ellipsis, the name would get forever cutoff. If a user has something like `two otters telling secrets on an iceberg or something else` and `two otters telling secrets on an iceberg or something else 2` they would know the difference. So maybe this ticket is actually obsolete now that the title isn't being used above? Or are you thinking the ellipsis would be clickable and would show the rest of the text somehow?

Reply:
I was more thinking the latter - since the ellipsis would say "hey, there's more copy here" and then either scrolling or clicking into it you get the chance to see it. But on second thought, there's nothing really getting cut off as is and you know the moment you scroll that the title's longer. None of our other inputs show an ellipsis if the content gets longer. I think this is obsolete, like you said! 

Let's confirm with design today in our sync.

Reply:
Just looping back to this post design sync - we all agree that this is obsolete since the dialog style has changed since this ticket was originally created. No need to update anything with an ellipsis, this is all within our existing patterns and expectations. 

Sorry for not catching the design change before throwing this into the sprint. I'm glad that in it you found the idea to update to a text area when the media's a video instead of a text input - so still a nice use of time.